### PR TITLE
doc(network-v2): fix section nesting levels

### PR DESCRIPTION
**Pull request by aciba90**
## Proposed Commit Message

```
doc(network-v2): fix section nesting levels

Place `set-name` and `wakeonlan` out of `match`, see [0]. Move the elements under the previous part one level up.

References:

[0] https://netplan.readthedocs.io/en/latest/netplan-yaml/#properties-for-physical-device-types
```

## Checklist

- [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/development/index.html)
- [ ] I have updated or added any [unit tests](https://cloudinit.readthedocs.io/en/latest/development/testing.html) accordingly
- [ ] I have updated or added any [documentation](https://cloudinit.readthedocs.io/en/latest/development/contribute_docs.html) accordingly

## Merge type

- [x] Squash merge using "Proposed Commit Message"
- [ ] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

File: doc/rtd/reference/network-config-format-v2.rst [hunk 1]
```diff
@@ -309,69 +309,69 @@ Example: ::
...
 ``parameters: <(mapping)>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
 
 Customisation parameters for special bonding options. Time values are
 specified in seconds unless otherwise specified.
 
 ``mode: <(scalar)>``
-^^^^^^^^^^^^^^^^^^^^
+--------------------
```
Comment (blackboxsw) [inline doc/rtd/reference/network-config-format-v2.rst:318]: `mode` through `learn-packet-interval` depth should be unchanged as they all live under parameters.

File: doc/rtd/reference/network-config-format-v2.rst [hunk 2]
```diff
@@ -468,70 +468,70 @@ Example: ::
...
 ``parameters: <(mapping)>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
 
 Customisation parameters for special bridging options. Time values are
 specified in seconds unless otherwise stated.
 
 ``ageing-time: <(scalar)>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
```
Comment (blackboxsw) [inline doc/rtd/reference/network-config-format-v2.rst:477]: `ageing-time` through `stp` should also remain unchanged. They live under `bridge:...:parameters:`

**Review by blackboxsw — CHANGES_REQUESTED** [review]
Thanks @aciba90 couple of corrections on the config keys under `parameters` for bridges and bonds. After that +1.

**Comment by aciba90** [comment]
> Thanks @aciba90 couple of corrections on the config keys under `parameters` for bridges and bonds. After that +1.

Thanks, good catch! I have amended it.

**Review by blackboxsw — APPROVED** [review]
Thanks for this @aciba90

**Pull request merged**